presto-product-tests/bin/run_on_docker.sh

```diff
@@ -137,6 +136,11 @@ fi
 TEMPTO_CONFIGURATION="${DOCKER_PRESTO_VOLUME}/presto-product-tests/conf/tempto/tempto-configuration.yaml"
 if [[ "$ENVIRONMENT" == "singlenode-kerberized" ]]; then
    TEMPTO_CONFIGURATION="${DOCKER_PRESTO_VOLUME}/presto-product-tests/conf/tempto/tempto-configuration-kerberized.yaml"
```

Comment:
Mount the required `tempto-configuration` in `docker-compose` instead of adding this `if`.